spacy_llm/tasks/span.py

```diff
@@ -47,16 +78,37 @@ def __init__(
         if self._prompt_examples:
             self._prompt_examples = self._check_label_consistency()
 
+    @property
+    def labels(self) -> Tuple[str, ...]:
+        return tuple(self._label_dict.values())
+
+    @property
+    def prompt_template(self) -> str:
+        return self._template
+
     def _check_label_consistency(self) -> List[SpanExample]:
         """Checks consistency of labels between examples and defined labels. Emits warning on inconsistency.
         RETURNS (List[SpanExample]): List of SpanExamples with valid labels.
         """
         assert self._prompt_examples
+        null_labels = {
+            self._normalizer(entity.label): entity.label
+            for example in self._prompt_examples
+            for entity in example.entities
+            if not entity.is_entity
+        }
+        if len(null_labels) > 1:
+            warnings.warn(
+                f"Negative examples contain multiple negative labels: {', '.join(null_labels.keys())}."
+            )
```

Comment:
I'm not sure I understand this check. Could you elaborate on why this is necessary?

Reply:
You have to have some kind of label for negative examples, we just use a random string like `none`. It might be a bit odd if someone were to use multiple different variants, so it probably makes sense to inform them.